File: 02-array-arraylist/Swap.java
import java.util.*;
public class Swap {
    public static void main(String[] args) {
        int[] arr = { 2, 23, 21, 8, 7, 3 };
        System.out.println(Arrays.toString(arr));
        swap(arr, 1, 4);
        System.out.println(Arrays.toString(arr));
    }
    
    static void swap(int[] arr, int i, int j) {
        int temp = arr[i];
        arr[i] = arr[j];
        arr[j] = temp;
    }
}

File: 02-array-arraylist/Swap.js
function swap(arr, i, j) {
  let temp = arr[i];
  arr[i] = arr[j];
  arr[j] = temp;
}

const arr = [2, 23, 21, 8, 7, 3];
console.log(arr);
swap(arr, 1, 5);
console.log(arr);
